services/nutanix-ai/metadata.yaml

```diff
@@ -54,10 +54,10 @@ overview: |-
       GPT-in-a-Box supports seamless integration with existing Nutanix products, thus providing a robust
       and compliant solution.
 
-  ## Documentation Link
+  ## More Information
   - [Nutanix AI Documentation](https://www.nutanix.com/solutions/ai)
```

Comment:
we wanted to add portal link here rite?

Reply:
Could you please elaborate and share which portal link here?

Reply:
I will add this in another PR.